Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,78 @@
+using Godot;
+
+public class PlayerCamera : Camera
+{
+    [Export]
+    private readonly float PlayerDistanceThreshold = 2f;
+    [Export]
+    private readonly float CursorDistanceThreshold = 1f;
+    [Export]
+    private readonly float OffsetZ = 25f;
+    [Export]
+    private readonly float Height = 50f;
+    [Export]
+    private readonly float LookRange = 5.0f;
+    [Export]
+    private readonly float FollowCursorSpeedFactor = 0.3f;
+    [Export]
+    private readonly float FollowPlayerSpeedFactor = 0.7f;
```

Comment:
If these must add up to 1, better to only have only one of them exported, while the other set to `1-xxx`, to avoid any unnecessary errors in the future.